utils/syncthing/files/etc/config/syncthing

```diff
@@ -0,0 +1,3 @@
+config syncthing 'syncthing'
+    option gui_address 'http://127.0.0.1:8384'
```

Comment:
What should be LAN address instead?

Reply:
I'd keep the default "secure", don't you think?

Reply:
Since the vast majority of devices doesn't have a screen or even output it doesn't seem like a very logical choice. In general we try to provide a good experience out of the box.
@neheb @BKPepe 
Comments?

Reply:
I have no idea if this binds to lo or if it doesn't bind to a specific iinterface. If the latter, meaningless as it would still be accessible through the LAN. If the former, your concern is correct.

If configurable, binding to lo and br-lan should be fine.

Reply:
It appears syncthing [binds to this address/port](https://docs.syncthing.net/users/guilisten.html) for its GUI (web) interface and REST API. (`127.0.0.1:8384` is the default, but they also assume you are using it from a desktop.)

I imagine most people will immediately change it to `0.0.0.0:8384` (following the documentation) or to br-lan, since they need to do more [configuration](https://docs.syncthing.net/intro/getting-started.html#configuring) to actually sync between devices.

Forcing most people to change it is not great, but having a secure (if useless) default is probably better.